src/golang/lib/workflow/operator/operator.go

```diff
@@ -25,17 +26,23 @@ type Operator interface {
 	JobSpec() job.Spec
 
 	// Launch kicks off the execution of this operator, using operator's job spec.
-	// Poll on `GetExecState()` afterwards to determine when this operator has completed.
+	// Use `Poll()` afterwards to determine when this operator has completed.
```

Comment:
Can you note that this sets the execution state of this operator to RUNNING? And that it doesn't update the execution state in the db.